### PR TITLE
Add ITREX unit test

### PR DESCRIPTION
**Pull request by XuehaoSun**
## Type of Change

Add ITREX unit test CI.

## How has this PR been tested?

CI

## Dependency Change?

no

### CODE REVIEW

File: .azure-pipelines/ut-itrex.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+pr:
+  autoCancel: true
+  drafts: false
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - neural_compressor
+      - test
```
Comment (chensuyue) [inline .azure-pipelines/ut-itrex.yml:12]: no need `test` path.

Comment (XuehaoSun) [reply .azure-pipelines/ut-itrex.yml]: fixed

File: .azure-pipelines/ut-itrex.yml [hunk 2]
```diff
@@ -9,10 +9,10 @@ pr:
   paths:
     include:
       - neural_compressor
-      - test
       - setup.py
       - requirements.txt
-      - .azure-pipelines/scripts/ut
+      - .azure-pipelines/scripts/ut/run_itrex.sh
+      - .azure-pipelines/ut-itrex.yml
     exclude:
```
Comment (chensuyue) [inline .azure-pipelines/ut-itrex.yml:16]: remove this part.

**Review by chensuyue — APPROVED** [review]

File: .azure-pipelines/ut-itrex.yml (hunk 2 above)
Comment (XuehaoSun) [reply .azure-pipelines/ut-itrex.yml]: fixed

**Pull request merged**